papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,584 @@
+However, this type of conditional workflow can be inefficient when sequential workflows are coupled with an HPC scheduler interface, because the user must log on to the HPC and submit the next operation after the previous operation is complete.
+The desire to submit large and long-running jobs to HPC schedulers encourages users to write large operation functions which are not modular and do not accurately represent the individual units of the workflow, thereby limiting **signac-flow**'s utility and reducing the readability of the workflow.
+
+Groups, implemented by the ``FlowGroup`` class and ``FlowProject.make_group`` method, allows users to combine multiple operations into a single entity that can be run or submitted.
+Submitting a group allows **signac-flow** to dynamically resolve preconditions and postconditions of operations as each operation is executed, making it possible to combine separate operations (e.g. for simulation and analysis and plotting) into a single submission script with the expectation that all will execute despite the dependencies between operations.
+Furthermore, groups are aware of directives and can properly combine the directives of their constituent operations to specify resources and quantities like walltime whether executing in parallel or serial.
+
+.. code-block:: python
+
+    from flow import FlowProject
+
+    new_group = FlowProject.make_group(name="new_group")
+
+    @new_group.with_directives(
+        {"ngpu": 2,
+         "walltime": lambda job: job.doc.hours_to_run})
+    @FlowProject.post.true("simulated")
+    @FlowProject.operation
+    def simulate(job):
+        # run simulation
+        job.doc.simulated = True
+
+    @new_group
+    @FlowProject.pre.after(simulate)
+    @FlowProject.post.true("analyzed")
+    @FlowProject.operation
+    def analzye(job):
+        # analyze simulation results
+        job.doc.analzyed = True
+
+Groups also allow for specifying multiple machine specific resources (CPU or GPU) with the same operation.
+An operation can have unique directives for each distinct group to which it belongs.
+By associating an operation's directives with respect to a specific group, groups can represent distinct compute environments, such as a local workstation or a remote supercomputing cluster.
+The below snippet shows an ``expensive_simulate`` operation which can be executed with three
+different directives depending on how it is written.
+If executed through ``cpu_env`` the operation will request 48 cores, if ``gpu_env`` 4 GPUs, if
+neither then it will request 4 cores.
+This represents the real use case where an user may want to run an operation locally (in this case
+without a group), or on a CPU or GPU focused HPC/workstation.
+
+.. code-block:: python
+
+    from flow import FlowProject
+
+    cpu_env = FlowProject.make_group(name="cpu")
+    gpu_env = FlowProject.make_group(name="gpu")
```

Comment:
is there a reason the name is different than the variable name here?

Reply:
not really... actually the name is a bit confusing.

Reply:
I changed the name to something I think to be more descriptive.